P5/Source/Guidelines/en/HD-Header.xml

```diff
@@ -1799,10 +1799,11 @@ more than one language, as in the following example:
       <specList>
         <specDesc key="unitDef"/>
         <specDesc key="unit"/>
-        <specDesc key="conversion" atts="formula fromUnit toUnit from to notBefore notAfter when"/> 
+        <specDesc key="conversion" atts="formula fromUnit toUnit from to notBefore notAfter when where"/> 
         <specDesc key="att.formula" atts="formula"/>
+        <specDesc key="att.locatable" atts="where"/>
       </specList>
-Within the <gi>unitDef</gi>, a <gi>conversion</gi> element may be used to store information relating to conversion between units. The <gi>conversion</gi> element holds a special pair of attributes, <att>fromUnit</att> and <att>toUnit</att>, which serve to indicate the direction of a calculation from one unit of measure (stored in <att>fromUnit</att>) to another (stored in <att>toUnit</att>). A mathematical calculation to define the relation between these units may be stored in <att>formula</att>, as shown in the following examples. The <att>formula</att> attribute takes a value expressed as an XPath expression, which means that division must be expressed with <q>div</q> so as not to be confused with the forward slash used in path navigation. 
+Within the <gi>unitDef</gi>, a <gi>conversion</gi> element may be used to store information relating to conversion between units. The <gi>conversion</gi> element holds a special pair of attributes, <att>fromUnit</att> and <att>toUnit</att>, which serve to indicate the direction of a calculation from one unit of measure (stored in <att>fromUnit</att>) to another (stored in <att>toUnit</att>). A mathematical calculation to define the relation between these units may be stored in <att>formula</att>, as shown in the following examples. The <att>formula</att> attribute takes a value expressed as an XPath expression, which means that division must be expressed with <q>div</q> so as not to be confused with the forward slash used in path navigation. The <gi>conversion</gi> element may also hold attributes to store dates and date ranges as well as location(s) that specify when and where a conversion formula was used.
```

Comment:
@sydb Here are responses with questions:
1. This is an opportunity to improve on the initial documentation for Would a reference to Michael Kay (2007) (in our BIBL) satisfy the need to provide more information on XPath expressions? It is what we provide in [teidata.xpath](https://www.tei-c.org/release/doc/tei-p5-doc/en/html/ref-teidata.xpath.html).

 I remember that we expected lots of division in conversion formulas, by the way, so that's why we highlighted the special use of "div" in XPath. That said, I like this rewriting of yours:
"The formula attribute takes a value expressed as an XPath expression, for which see . Among other things, this means that division must be expressed with div so as not to be confused with the forward slash used in path navigation."

2. Surveying the Guidelines and Specs for how we describe elements in relation to attributes, apparently it's more typical for elements to "use" attributes to say something specific. Or they "supply" information. So, how about this revision to that last sentence?
 
"Attributes on the <gi>conversion</gi> element may also supply dates and date ranges as well as location(s) that specify when and where a conversion formula was used."

Reply:
(updated this a little for clarity): read on GitHub.

Reply:
1. Yes, could certainly use the XSLT spec, which is what “Kay” refers to from teidata.xpath. As you can now see (since I have edited my post) I had suggested the XPath spec itself rather than the XSLT spec, which I think makes more sense here (and maybe in teidata.xpath, too). But I think a further tweak to the wording is needed, thus: “`The <att>formula</att> attribute takes a value expressed as an XPath expression, for which see <ptr target="#XPATH2"/>. Among other things, this means that division must be expressed with <q>div</q> as opposed to the more common forward slash (<q>/</q>, U+002F), so as not to be confused with path navigation.`”.

Reply:
Check.